src/app/public/modules/phone-field/phone-field.component.html

```diff
@@ -0,0 +1,45 @@
+<div
+  class="sky-form-group sky-input-group"
+>
+  <div
+    class="sky-input-group-btn sky-input-group-country-btn"
+  >
+    <sky-dropdown
+      [disabled]="disabled"
+      [label]="'skyux_phone_field_dropdown_label' | skyLibResources"
+      [title]="selectedCountry.name + ' +' + selectedCountry.dialCode"
+    >
+      <sky-dropdown-button>
+        <div
+          [className]="'iti-flag ' + selectedCountry.iso2"
+        >
+        </div>
+      </sky-dropdown-button>
+      <sky-dropdown-menu>
+        <sky-dropdown-item *ngFor="let country of countryData; let i = index"
+          [ngClass]="{
+            'sky-selected-country': i === 0,
+            'sky-default-country': country === defaultCountryData
```

Comment:
Should this logic be in the component to be better accessible for tests?

Reply:
I personally don't think so. These are two seperate one line logic checks and the first relies on data from the template. The only thing that can really be tested around this logic is visual and they are covered in visual tests so I think we are good to leave them here.